3.4/Dockerfile.rhel7

```diff
@@ -32,13 +32,14 @@ LABEL summary="$SUMMARY" \
       version="3.4" \
       release="1"
 
-# To use subscription inside container yum command has to be run first (before yum-config-manager)
-# https://access.redhat.com/solutions/1443553
+# We need to call 2 (!) yum commands before being able to enable repositories properly
+# This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1479388
 RUN yum repolist > /dev/null && \
+    yum install -y yum-utils && \
+    yum-config-manager --disable \* &> /dev/null && \
+    yum-config-manager --enable rhel-7-server-rpms && \
     yum-config-manager --enable rhel-server-rhscl-7-rpms && \
```

Comment:
In version 3.4 the ordering of the enabled repositories is different than in 2.7 and 3.5. It might be worth fixing that so the files are as similar as possible.